### PR TITLE
Add TKL keymap for Frosty_Flake

### PR DESCRIPTION
**Pull request by jasonm23**
Provide simpler/explicit support for TKL keymaps for the Frosty Flake

Includes a simple example keymap and update to the Frosty Flake README.md

### CODE REVIEW

File: keyboards/frosty_flake/readme.md [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
+## 104 and 87 layout support
+
+Support for both 104 key and 87 key layouts is provided. See the
+keymaps `default` (104) and `tkl` (87) for example layouts.
+
+**Keyboard Maintainer:** QMK Community
+
+**Hardware Supported:** Frosty Flake
+
+**Hardware Availability:** https://1upkeyboards.com/qfr-frosty-flake-controller.html
```
Comment (skullydazed) [inline keyboards/frosty_flake/readme.md]: Our readme standard puts these on 3 consecutive lines without the bolding. You should put 2 spaces at the end so the line break makes it to the html. Could you please update this to comply with that?

**Comment by jasonm23** [comment]
hi @skullydazed please take a look now, should be good.

**Comment by skullydazed** [comment]
Perfect, thanks!

**Comment by jasonm23** [comment]
Awesome, have a 🏅

**Pull request merged**